src/AzureExtension/DataModel/DataObjects/Identity.cs

```diff
@@ -145,19 +163,19 @@ private static Identity CreateFromIdentity(Microsoft.VisualStudio.Services.Ident
         };
     }
 
-    public static Identity AddOrUpdateIdentity(DataStore dataStore, Identity identity, bool isDeveloper = false)
+    public static Identity AddOrUpdateIdentity(DataStore dataStore, Identity identity, string developerLoginId = "")
     {
         // Check for existing Identity data.
         var existingIdentity = GetByInternalId(dataStore, identity.InternalId);
         if (existingIdentity is not null)
         {
             identity.Id = existingIdentity.Id;
 
-            // If this is a developer, set to developer, but do not set to false.
+            // If this is a developer, set to developer, but do not set it.
```

Comment:
Can you be explicit about what "it" is here?

Reply:
The value was changed from boolean to a string so the meaning was lost, the comment doesn't even make sense to me anymore so I removed it.